### ISSUE TITLE
theme.json blocks styles duplicates css

### ISSUE THREAD

**Issue by mr660** [WordPress/gutenberg#65535]
### Description

I'm setting default color for block `core/separator` using `css` property and i noticed duplication of inline css.

theme.json
```
"styles":{
  "blocks": {
        "core/separator": {
          "css": "color:var(--wp--preset--color--rich-electric-blue)"
        }
  }
}
```

Result:
![image](https://github.com/user-attachments/assets/29b650d8-b32f-4325-916a-c00b710dc0e1)

When i remove this css:
![image](https://github.com/user-attachments/assets/657e8aa5-29b2-4296-ac19-939c0e0c5644)

### Step-by-step reproduction instructions

- set up theme.json like above

### Environment info

- WP 6.6.2

### Please confirm that you have searched existing issues in the repo.

- [X] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [X] Yes

**Comment by Mayank-Tripathi32** [comment]
Hi @mr660,

I was able to replicate the issue with version 6.6.2 by following the steps you provided.

![Image](https://github.com/user-attachments/assets/9cd83d33-2cb5-430a-b23e-24e29540124e)

However, it seems to be working fine with 6.8-alpha-59374:

![Image](https://github.com/user-attachments/assets/68b97b52-d7b4-4402-ab9d-354e6ae7701f)

I believe this might have been resolved in the 6.7 release, but I’ll investigate further to confirm.

**Comment by youknowriad** [comment]
Closing as it seems fixed.
